source/configuration/modules/mmkubernetes.rst

```diff
@@ -0,0 +1,306 @@
+Kubernetes Metadata Module (mmkubernetes)
+=========================================
```

Comment:
Not a requirement, but to match the syntax that we're converting other module docs to would you mind updating the header like so?

```diff
+*****************************************
 Kubernetes Metadata Module (mmkubernetes)
-=========================================
+*****************************************
```